test/testgroups/operatorsourcetests.go

```diff
@@ -32,4 +32,5 @@ func OperatorSourceTestGroup(t *testing.T) {
 	// Run the test suites.
 	t.Run("opsrc-creation-test-suite", testsuites.OpSrcCreation)
 	t.Run("csc-target-namespace-test-suite", testsuites.CscTargetNamespace)
+	t.Run("csc-packages-test-suite", testsuites.CscPackages)
```

Comment:
Why do we need a separate test suite for this test?

Reply:
target namespace has one - im just following the implied template.

Reply:
I support having a separate test suite here.